src/main/java/org/mockito/Mockito.java

```diff
@@ -2702,4 +2696,17 @@ static MockitoDebugger debug() {
     public static MockitoFramework framework() {
         return new DefaultMockitoFramework();
     }
+
+    /**
+     * Allows to manage Mockito strictness (debugging, cleaner tests) without JUnit.
+     * <p>
+     * TODO javadoc
+     *
+     * @param testClassInstance
+     * @param strictness
+     */
+    @Incubating
+    public static MockitoMocking startMocking(Object testClassInstance, Strictness strictness) {
+        return new DefaultMockitoMocking(testClassInstance, strictness, new ConsoleMockitoLogger());
+    }
```

Comment:
Absolutely we need to be cautious about adding more public method to Mockito class because it is so big. On the other hand, our users expect to find new API on Mockito class, it is a main entry to our entire API. MockitoAnnotations is an odd exception that we can live with because everybody is so used to it ;)

An alternative approach is to new create classes like MockitoSession, and place new public static methods there. However, this would fragment the API and would make it harder to use. Having lots of methods on Mockito is not convenient. However, having several static classes with API entry points is even more inconvenient. It makes the API less discoverable.

Given decent use case and no other sensible host, I'm ok with new method on Mockito class. 

Given @TimvdLippe vote on this direction, I'm going ahead with Mockito.mockitoSession() in this PR.

Thanks everybody for great feedback!